docs/ops/movement/ShuffleChannels_1.md

```diff
@@ -8,43 +8,57 @@
 
 **Short description**: *ShuffleChannels* permutes data in the channel dimension of the input tensor.
 
+**Detailed description**:
+
+Input ND tensor of `data_shape` is always interpreted as 4D tensor with the following shape:
+
+    dim 0: data_shape[0] * data_shape[1] * ... * data_shape[axis-1] (or 1 if axis == 0)
+    dim 1: group
+    dim 2: data_shape[axis] / group
+    dim 3: data_shape[axis+1] * data_shape[axis+2] * ... * data_shape[data_shape.size()-1] (or 1 if axis points to last dimension)
```

Comment:
I think that this text will be rendered ugly. Can we wrap the text with some style modifier for pretty displaying?

Reply:
The tab on the begining of each line work as style modifier and make it a separate pseudo-code like block.
I think it's not so ugly now;)


![image](https://user-images.githubusercontent.com/20084299/121190995-d9703f00-c86b-11eb-8b9c-53d507f5e231.png)